ballerina/soap.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/mime;
+
+# Object for the basic SOAP client endpoint.
+public client class BasicClient {
```

Comment:
Are we supporting SOAP 1.1 and SOAP 1.2 through this client? I don't see a configuration for that

Reply:
As per the design[[1]](https://github.com/ballerina-platform/ballerina-standard-library/issues/4618#issuecomment-1639234830), the basic client only supports soap 1.1.

[1] https://github.com/ballerina-platform/ballerina-standard-library/issues/4618#issuecomment-1639234830

Reply:
Updated in https://github.com/ballerina-platform/module-ballerina-soap/pull/82